spec/requests/user_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+RSpec.describe 'USER', type: :request do
+  describe 'user#index' do
+    before(:each) { get user_index_path }
+
+    it 'server return 200 then server is ok' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'it shows user#index route and template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'user#index content is a h1 tag with index user' do
+      expect(response.body).to include('<h1>index user</h1>')
+    end
+  end
```

Comment:
- [ ] This makes your test failing, kindly target the text that is included in the body instead of using 

`(<h1>index user </h1>) `

use ('index user') to target that text.

Reply:
It was not failing here tho